dist/main.css

```diff
@@ -61,3 +61,28 @@ body {
     transform: rotate(360deg);
   }
 }
+
+a.dropdown-toggle:after {
+  content: none !important;
+}
+
+.navbar-collapse {
+  font-size: 1.25rem;
+}
+
+@media (min-width: 992px) {
+  .navbar-expand-lg .navbar-nav .dropdown-menu {
+    border: none;
+    background-color: #f8f9fa !important;
+  }
+
+  .dropdown-item {
+    padding: .5rem 1.5rem;
+    border-bottom: 1px solid rgba(0,0,0,.3);
+  }
+
+  .dropdown-item:last-child {
+    border-bottom: none;
+    padding-bottom: 0;
```

Comment:
Instead of overriding the styles here, I think the default looks pretty clean: 
![image](https://user-images.githubusercontent.com/8093386/101199119-4d116500-3619-11eb-8395-d076135d2af5.png)

Is there anything prohibiting us from doing it this way?
https://react-bootstrap.github.io/components/navbar/#navbars-overview